### DISCUSSION TITLE
Is there any way to set Two way Sync?

### DISCUSSION THREAD

**Discussion by muuky** [remotely-save/remotely-save#300] [Q&A]
In my case, I often move from one folder to another or delete files.
Also, using Obsidian on several devices via Webdav from my Synology NAS.

So, if I move a file to another folder on your main device, duplicate files will be created on the sub device.

The reason why this happens is that files that have already been deleted from the Webdav server remain in other sub devices.
Since there are files on the sub device that are not on the Webdav server, Remotely Save Plugin recognizes them as new files and uploads them to the Webdav server.

For example, When the folder structure of the main device and sub device is as follows
```
Main Device
  - Notes
    - 20230225 Note.md
    - 20230224 Note.md
```

```
Sub Device
  - Notes
    - 20230225 Note.md
    - 20230224 Note.md
```

If I move the file on the main device like this
```
Main Device
  - Notes
    - 20230225 Note.md
  - Old Notes
    - 20230224 Note.md
```

The folder structure of the sub device became like this

```
Sub Device
  - Notes
    - 20230225 Note.md
    - 20230224 Note.md
  - Old
    - 20230224 Note.md
```

In my opinion, it is because the Remote save plugin does not have deletion function.
Is there any way to set Two-way Sync like Synology Drive Client?

**Comment by fyears** [comment]
a new version two weeks later should fix this
